orca-peering/src/main/kotlin/com/netflix/spinnaker/orca/peering/PeeringAgent.kt

```diff
@@ -138,8 +138,8 @@ class PeeringAgent(
     log.debug("Starting completed $executionType copy for peering with $executionType updatedAfter=$updatedAfter")
 
     // Compute diff
-    val completedPipelineKeys = srcDB.getCompletedExecutionIds(executionType, peeredId, updatedAfter)
-    val migratedPipelineKeys = destDB.getCompletedExecutionIds(executionType, peeredId, updatedAfter)
+    val completedPipelineKeys = srcDB.getCompletedExecutionIds(executionType, peeredId, true, updatedAfter)
+    val migratedPipelineKeys = destDB.getCompletedExecutionIds(executionType, peeredId, false, updatedAfter)
```

Comment:
Good catch, but I'm a little uncomfortable with an unassuming boolean getting passed to the same function here, the two function calls read almost the same but behave crucially differently. Since the boolean changes the semantics of `getCompletedExecutionIds`, I think it would make sense to create two higher-level functions with more descriptive names. For instance, something like:

```
getCompletedExecutionIdsToMigrate(executionType, peeredId, updatedAfter) -> calls with allowNullPartition = true
getMigratedExecutionIds(executionType, peeredId, updatedAfter) -> calls with allowNullPartition = false
```

Reply:
yeah, i agree. What do you think about making two separate calls: e.g.
```java
srcDB.getCompletedExecutionIds(executionType, peeredId, updatedAfter)
srcDB.getCompletedExecutionIds(executionType, null, updatedAfter)
```
and concatting the result - that i think makes it obvious what's up and doesn't leak this `null` parition history into the sql access layer?